README.md

```diff
@@ -190,9 +193,82 @@ time. In theory, it should work with minimal adaptations.
 
 Anyway, you need to have hardware 3D OpenGL support to run Blackvoxel.
 
+###### This will compile the last version:
+
+### Ubuntu
+
+1. Open a terminal type: "term" in ubuntu lancher or press 'ctrl-alt-t'
+2. Enter the following commands:
+```
+git clone https://github.com/rajdakin/Blackvoxel.git
```

Comment:
Pointing to your own fork?
This pull request is obviously not ready to be merge back upstream.